### PR TITLE
POH: Mythical Cape icon

### PR DESCRIPTION
**Pull request by JZomerlei**
image

I am coming up blank with my pixel art.. Some one who is much better can take it from here :)

### CODE REVIEW

**Comment by raiyni** [comment]
Can you post a screenshot of this not zoomed in?

**Comment by JZomerlei** [comment]
@raiyni

**Comment by TheStonedTurtle** [comment]
@JZomerlei that looks like a different icon than your first post, it has brown under the hood where it was white before. If this was intentional you may want to update the zoomed-in picture as well

**Comment by JZomerlei** [comment]
Oh shoot you're right! I will revert to that version... Not sure what happened there.

File: runelite-client/src/main/java/net/runelite/client/plugins/poh/PohIcons.java [hunk 1]
```diff
@@ -86,6 +86,8 @@
 	),
 	DIGSITEPENDANT("digsitependant",
 		DIGSITE_PENDANT, DIGSITE_PENDANT_33417, DIGSITE_PENDANT_33418, DIGSITE_PENDANT_33420
+	),
+	MYTHICALCAPE("mythicalcape", MYTHICAL_CAPE, MOUNTED_MYTHICAL_CAPE
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/poh/PohIcons.java]: can you make this code style consistent with the rest of the file?

Comment (JZomerlei) [reply runelite-client/src/main/java/net/runelite/client/plugins/poh/PohIcons.java]: Glory, Pools, Repair, etc are all one liners as well.

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/poh/PohIcons.java]: Move the closing paren from the next line to the same line so this line reads:

```
	MYTHICALCAPE("mythicalcape", MYTHICAL_CAPE, MOUNTED_MYTHICAL_CAPE),
```

**Comment by JZomerlei** [comment]
```
	DIGSITEPENDANT("digsitependant",
		DIGSITE_PENDANT, DIGSITE_PENDANT_33417, DIGSITE_PENDANT_33418, DIGSITE_PENDANT_33420
	);
```

The closing ) was on the next line, just like how the ), is on the next line for all the others. I am being consistent I think.

**Comment by Nightfirecat** [comment]
For one-line definitions (which this one is), the closing paren is on the same line. For multi-line definitions (which this one is not), the closing paren is on a new line.

**Comment by JZomerlei** [comment]
Okay. I’ll change it when I get home.

**Comment by JZomerlei** [comment]
Fixed.

**Pull request merged**